Main/Source/cmdcraft.cpp

```diff
@@ -1592,6 +1731,25 @@ struct srpWorkBench : public srpOltBASE{
     return srpOltBASE::work(rpd);
   }
 };srpWorkBench rpWorkBench;
+struct srpTWorkBench : public srpOltBASE{
+  virtual bool spawnCfg(recipedata& rpd){
+    rpd.otSpawnType=CTT_FURNITURE;
+    rpd.otSpawnCfg=TAILORING_BENCH;
+    return true;
+  }
+
+  virtual void fillInfo(){
+    init("build","a tailoring workbench");
```

Comment:
Maybe change normal workbench to "crafting workbench" then, to make it clear in the F menu how are the two workbenches different?

Reply:
may be "smithing bench" but I read it is more related to metal work, "crafting workbench" is more generic indeed and may be clearer

Reply:
Yeah, "smithing" is metal work and we already have an anvil for that. :)